extensions-core/stats/src/test/java/org/apache/druid/query/aggregation/variance/VarianceObjectVectorAggregatorTest.java

```diff
@@ -47,6 +47,13 @@ public class VarianceObjectVectorAggregatorTest extends InitializedNullHandlingT
       new VarianceAggregatorCollector(2, 183, 1984.5)
   };
   private static final boolean[] NULLS = new boolean[]{false, false, true, true, false};
+  private static final Object[] OBJECTS = new Object[]{
```

Comment:
can you leave a comment stating that
"this variable is same as VALUES but with a different type. This is used in *compatability tests to verify that VarianceObjectVectorAggregator can handle object arrays as well"

Reply:
ok, this is a good suggestion.